modules/server/src/main/scala/navigate/server/tcs/TcsBaseControllerSim.scala

```diff
@@ -52,4 +52,7 @@ class TcsBaseControllerSim[F[_]: Applicative] extends TcsBaseController[F] {
 
   override def rotIaa(angle: Angle): F[ApplyCommandResult] =
     Applicative[F].pure(ApplyCommandResult.Completed)
+
+  override def oiwfsProbeTracking(config: TrackingConfig): F[ApplyCommandResult] =
+    Applicative[F].pure(ApplyCommandResult.Completed)
```

Comment:
`ApplyCommandResult.Completed.pure[F]` just a comment, dont worry if you prefer the other style